README.md

```diff
@@ -73,6 +113,17 @@ skeleton directory. To build that project, do something like:
 make -C build
 ````
 
+## Building
+
```

Comment:
Maybe we should point out the minimal set of packages one would need to have installed?

Reply:
Added a note about the installation, including a reference to the bug in the debian/ubuntu packages and the work around we used in the travis file.